GoogleAnalyticsEventForwarder.js

```diff
@@ -521,13 +521,21 @@
                         })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                     }
 
+                    var fieldsObject = {
+                        trackingId:forwarderSettings.apiKey,
+                        name: trackerId
+                    };
+
                     if (forwarderSettings.useLocalhostCookie == 'True') {
-                        ga('create', {
-                            trackingId: forwarderSettings.apiKey,
-                            cookieDomain: 'none',
-                            name: trackerId
-                        });
+                        fieldsObject.cookieDomain = 'none';
+
+                        ga('create', fieldsObject);
+                    } else if (forwarderSettings.useAmpClientId == 'True') {
```

Comment:
Is it legal to set both useLocalhostCookie and useAmpClientId? maybe you could just build up the `fieldsObject` and have a single `ga('create', fieldsObject)` vs the three now?

Reply:
ah yep